datasets/movielens/data.json

```diff
@@ -0,0 +1,244 @@
+{
+  "@context": {
+    "@vocab" : "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/"
+  },
+  "@type": "Dataset",
+  "@language": "en",
+  "name": "Movielens-25M",
+  "url": "https://grouplens.org/datasets/movielens/25m/",
+  "description": "MovieLens 25M movie ratings. Stable benchmark dataset. 25 million ratings and one million tag applications applied to 62,000 movies by 162,000 users. Includes tag genome data with 15 million relevance scores across 1,129 tags. Released 12/2019",
+  "distribution": [
+    {
+      "name": "ml-25m-archive",
+      "@type": "FileObject",
+      "contentUrl": "https://files.grouplens.org/datasets/movielens/ml-25m.zip",
+      "sha256": "6b51fb2759a8657d3bfcbfc42b592ada",
+      "encodingFormat": "application/zip"
+    },
+    {
+      "name": "ratings-table",
+      "@type": "FileObject",
+      "containedIn": "ml-25m-archive",
+      "contentUrl": "ratings.csv",
+      "encodingFormat": "text/csv"
+    },
+    {
+      "name": "movies-table",
+      "@type": "FileObject",
+      "containedIn": "ml-25m-archive",
+      "contentUrl": "movies.csv",
+      "encodingFormat": "text/csv"
+    },
+    {
+      "name": "tags-table",
+      "@type": "FileObject",
+      "containedIn": "ml-25m-archive",
+      "contentUrl": "tags.csv",
+      "encodingFormat": "text/csv"
+    },
+    {
+      "name": "links-table",
+      "@type": "FileObject",
+      "containedIn": "ml-25m-archive",
+      "contentUrl": "links.csv",
+      "encodingFormat": "text/csv"
+    },
+    {
+      "name": "genome-scores-table",
+      "@type": "FileObject",
+      "containedIn": "ml-25m-archive",
+      "contentUrl": "genome-scores.csv",
+      "encodingFormat": "text/csv"
+    },
+    {
+      "name": "genome-tags-table",
+      "@type": "FileObject",
+      "containedIn": "ml-25m-archive",
+      "contentUrl": "genome-tags.csv",
+      "encodingFormat": "text/csv"
+    }
+  ],
+  "recordSet": [
+    {
+      "name": "movies",
+      "@type": "ml:RecordSet",
+      "source": "#{movies-table}",
+      "key": "#{movie_id}",
+      "field": [
+        {
+          "name": "movie_id",
+          "@type": "ml:Field",
+          "dataType": "Integer",
+          "source": "#{movies-table/movieId}"
+        },
+        {
+          "name": "title",
+          "@type": "ml:Field",
+          "description": "The title of the  movie with the release year in parentheses.",
+          "dataType": "Text",
```

Comment:
I'm not very familiar with csvw, is the distinction between `"dataType": "Text"` and `"dataType": "String"` deliberate?
They are both used in these datasets.